pages/test_analytics/flaky_test_assignment.md

```diff
@@ -0,0 +1,25 @@
+# Flaky test assignment
+
+Customers on the [Business and Enterprise plans](https://buildkite.com/pricing) can assign flaky tests to [Teams](/docs/team_management/permissions).
```

Comment:
I see on that pricing page the plan names are 'pro' and 'enterprise'. Should we use the name 'pro' instead of 'business'?

Reply:
Ah yep, I'd just copied it from the Teams docs, have updated.